guides/source/active_record_migrations.md

```diff
@@ -632,8 +633,8 @@ actions automatically. Below are some of the actions that `change` supports:
 * [`rename_index`][]
 * [`rename_table`][]
 
-[`change_table`][] is also reversible, as long as the block does not call `change`,
-`change_default` or `remove`.
+[`change_table`][] is also reversible, as long as the block does not call irreversible operation
+which does not correspond to the operations listed above.
```

Comment:
`change_default` or `remove` may be reversible if proper option is given. Wording can be improved though.